Riot/Assets/it.lproj/Vector.strings

```diff
@@ -358,7 +359,7 @@
 "settings_copyright" = "Copyright";
 "settings_term_conditions" = "Termini e condizioni";
 "settings_privacy_policy" = "Privacy Policy";
-"settings_acknowledgement" = "Acknowledgement";
+"settings_acknowledgement" = "Credits";
```

Comment:
The change is also in en.lproj but not the other languages.  Is that intentional?

Reply:
Updated the branch to revert the accidental change in the Italian file, and to make sure that the settings_acknowledgement string reference is in all language files with the original 'Acknowledgement' value.